docs/network-flow-visibility.md

```diff
@@ -327,6 +339,28 @@ flow-aggregator.conf: |
 
     # TLS min version from: VersionTLS10, VersionTLS11, VersionTLS12, VersionTLS13.
     #tlsMinVersion:
+  
+  # clickHouse contains ClickHouse related configuration options.
```

Comment:
you need yo update the text above to match the new config

Reply:
updated the whole section to reflect the latest conf format.